src/pages/Home/Content/index.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from 'react';
+import Product from '../../../components/Product';
+
+import { Row, Col } from 'antd';
+
+import './style.scss';
+import { useSelector } from 'react-redux';
+
+const title = [
+  {
+    name: 'cafe',
+  },
+  {
+    name: 'mắt chim',
+  },
+  {
+    name: 'coolmax',
+  },
+  {
+    name: 'airy cool',
+  },
+  {
+    name: 'vỏ hàu',
+  },
+  {
+    name: 'cool compact',
+  },
+  {
+    name: 'airmax',
+  },
+  {
+    name: 'askin',
+  },
+];
+
+const Content = () => {
+  const { products } = useSelector((state) => state.productReducer);
+  const [active, setActive] = useState('cafe');
```

Comment:
```suggestion
  const [active, setActive] = useState(title[0]);
```
Nhỡ có đổi bên trong title thì cũng k sao